packages/scss/src/components/_actionIcon.scss

```diff
@@ -10,11 +10,24 @@
 	width: 2rem;
 
 	&:hover {
-		background: _component("actionIcon.hover.background");
-		color: darken(_color("text.default"), 25%);
+		&:not([disabled]) {
```

Comment:
Yes, good point. You should add a class "disabled" too, because all elements shouldn't have a "disabled" attribute